content/en/blog/2022/gateway-api-beta/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Extending Gateway API support in Istio"
+publishdate: 2022-07-13
+attribution: "Craig Box (Google)"
+keywords: [traffic-management,gateway,gateway-api,api,gamma,sig-network]
+---
```

Comment:
We also typically have a description here for the blog as well. I notice that this and the CNCF are two that don't for 2021 and 2022.